MMDevice/MMDevice.h

```diff
@@ -205,6 +208,104 @@ namespace MM {
    };
 
 
+   ////////////////////////////
+   ///// Standard properties
+   ////////////////////////////
```

Comment:
Much of the machinery here (the parts that are not intended for use by user device code) can be placed in namespace `MM::internal` to make this clear.